### ISSUE TITLE
is this used anywhere?

### ISSUE THREAD

**Issue by BogumilBar** [Taxijakt-AB/provider-taxistockholm-v2#1]
https://github.com/Taxijakt-AB/provider-taxistockholm-v2/blob/310fab614aa6880424d850615645582545168551/webapp/controllers/api-controller.ts#L85

**Comment by duffman** [comment]
No, it is not at the moment. It should be exposed once I figure a few things out first
